modules/mobianRtdProvider.js

```diff
@@ -1,17 +1,30 @@
+/**
+ * This module adds the Mobian RTD provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ * @module modules/anonymisedRtdProvider
```

Comment:
are you from idward / anonymised? do you mean to import their type?

Reply:
no -- that's embarrassing. I've had trouble getting the build to include the real time data module without including it explicitly, and hoped that typing would help. Will remove...

Reply:
Add yourself to  .submodules.js

Reply:
https://github.com/prebid/Prebid.js/blob/dc6c0c35ead5f44f3fc35852a84956a6c9036bed/modules/.submodules.json#L88